Sources/Chat/ProtocolServices/Common/MessagingService.swift

```diff
@@ -52,7 +53,8 @@ class MessagingService {
     private func setUpRequestHandling() {
         networkingInteractor.requestPublisher.sink { [unowned self] subscriptionPayload in
             switch subscriptionPayload.request.params {
-            case .message(let message):
+            case .message(var message):
+                message.topic = subscriptionPayload.topic
```

Comment:
Why we need to do that? Is it possible to avoid struct mutation?

Reply:
`wc_chatMessage` does not contain topic but when we query messages from storage it is convenient to distinguish them by topic.
maybe some storage improvement could allow to avoid this but instant sync with kotlin was a priority
will take this into consideration in next storage PR